src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting Started
+description: Learn how to set up and start using Daytona.
+sidebar:
+  label: Getting Started
+---
+
+import Aside from '@components/Aside.astro'
+import DocumentList from "@components/DocumentList.astro";
+import DocumentListItem from "@components/DocumentListItem.astro";
+import CardGrid from '@components/cards/CardGrid.astro';
+import TitleCard from '@components/cards/TitleCard.astro';
+import bitbucketLogo from '@assets/index/icons/Bitbucket.svg?raw';
+import githubLogo from '@assets/index/icons/GitHub.svg?raw';
+import gitlabLogo from '@assets/index/icons/GitLab.svg?raw';
+
+This section will guide you through the initial steps required to set up and start using Daytona effectively. This guide will ensure that you are well-equipped to manage and deploy your development environments using Daytona. It covers the essential steps, from installation to the creation of your first Workspace, providing clear instructions to help you get up and running quickly.
+
+## Before You Begin
+
+Before diving into the installation and usage of Daytona, it is crucial to ensure that your system meets the necessary prerequisites and that you have all the required tools and configurations in place. This section outlines the key requirements and preparatory steps you should take before getting started with Daytona.
+
+### System requirements
+
+Daytona supports multiple operating systems and architectures. Ensure your system matches one the following specifications:
+
+| **Operating Systems**                         | **Architectures**                                   |
+|-----------------------------------------------| ----------------------------------------------------|
+| [Linux](installation/installation#linux)      | `x86_64` or `AArch64`                               |
+| [macOS](/installation/installation#macos)     | `x86_64 (Intel-based)` or `AArch64 (Apple Silicon)` |
+| [Windows](/installation/installation#windows) | `x86_64` or `AArch64`                               |
+
+### Dependencies
+
+Daytona relies on specific dependencies to function correctly. Ensure that these dependencies are installed and properly configured on your system:
+
+| **Dependency**            | **Description**                                                                                 |
+|---------------------------|-------------------------------------------------------------------------------------------------|
+| Docker                    | Daytona uses Docker to create and manage isolated development environments.                     |
+| Git                       | Daytona requires Git to clone repositories and manage version control operations.               |
```

Comment:
@stefanicjuraj i dont think you need git installed @Tpuljak can you confim?

Reply:
You don't need it. We use our own git for clone operations